elliott/elliottlib/cli/find_bugs_golang_cli.py

```diff
@@ -0,0 +1,276 @@
+                 f'and labels = "SecurityTracking" '
+                 f'and "Target Version" in ({tr})')
+
+        bugs: List[JIRABug] = self.jira_tracker._search(query, verbose=self._runtime.debug)
+        logger.info(f"Found {len(bugs)} bugs")
+        bugs_by_component = {}
+        for b in sorted(bugs, key=lambda b: b.id):
+            component = b.whiteboard_component
+            if component not in bugs_by_component:
+                bugs_by_component[component] = []
+            bugs_by_component[component].append(b.id)
+        logger.info(f"Bugs by component: {bugs_by_component}")
+
+        self.flaw_bugs: Dict[int, BugzillaBug] = {}
+        fixed_bugs, unfixed_bugs = [], []
+        for bug in bugs:
+            component = bug.whiteboard_component
+            logger.info(f"{bug.id} has security component: {component}")
+
+            tracker_fixed_in = self.tracker_fixed_in(bug)
+            if not tracker_fixed_in:
+                self._logger.warning(
+                    f"Could not determine fixed in versions for {bug.id}. Ignoring it for now")
+                continue
+            logger.info(f"{bug.id} is fixed in: {tracker_fixed_in}")
+
+            fixed, comment = False, ''
+            if component == constants.GOLANG_BUILDER_CVE_COMPONENT:
+                fixed, comment = await self.is_fixed_golang_builder(bug, tracker_fixed_in)
+            else:
+                fixed, comment = await self.is_fixed_rpm(bug, tracker_fixed_in, component)
+
+            if fixed:
+                message = f"{bug.id} is determined to be fixed."
+                green_print(message)
+                if self.update_tracker:
+                    comment = f"{comment} {message}"
+                    self.jira_tracker.add_comment(bug.id, comment, private=True, noop=self.dry_run)
+                fixed_bugs.append(bug.id)
+            else:
+                unfixed_bugs.append(bug.id)
+
+        if fixed_bugs:
+            green_print(f'Fixed bugs: {sorted(fixed_bugs)}')
+        if unfixed_bugs:
+            red_print(f'Not fixed / unsure bugs: {sorted(unfixed_bugs)}')
+
+
+@cli.command("find-bugs:golang", short_help="Find golang related bugs")
+@click.option("--pullspec", default=None,
+              help="Pullspec of release payload to check against. If not provided, latest accepted nightly will be used")
+@click.option("--update-tracker",
+              is_flag=True,
+              default=False,
+              help="Comment on tracker with analysis and result")
+@click.option("--dry-run",
+              is_flag=True,
+              default=False,
+              help="Don't change anything")
+@click.pass_obj
+@click_coroutine
+async def find_bugs_golang_cli(runtime: Runtime, pullspec: str, update_tracker: bool, dry_run: bool):
+    """Find golang bugs in jira and determine if they are fixed or not
+
+    $ elliott -g openshift-4.14 --assembly 4.14.8 find-bugs:golang --comment
```

Comment:
Would love to see an explanation in human words what to expect that would happen.